lib/improver/tests/psychrometric_calculations/psychrometric_calculations/test_FallingSnowLevel.py

```diff
@@ -103,90 +103,130 @@ def setUp(self):
                                              [1.0, np.nan, 2.0],
                                              [1.0, 2.0, np.nan]])
         self.orog = np.ones((3, 3))
-        self.land_sea = np.ones((3, 3))
         self.highest_wb_int = np.ones((3, 3))
         self.highest_height = 300.0
 
     def test_basic(self):
-        """Test method returns an array with correct data"""
+        """Test fills in missing data with orography + highest height"""
         plugin = FallingSnowLevel()
+        self.highest_wb_int[1, 1] = 100.0
         expected = np.array([[1.0, 1.0, 2.0],
-                             [1.0, 1.5, 2.0],
+                             [1.0, 301.0, 2.0],
                              [1.0, 2.0, 2.0]])
-        result = plugin.fill_in_missing_data(self.snow_level_data,
-                                             self.orog, self.land_sea,
-                                             self.highest_wb_int,
-                                             self.highest_height)
-        self.assertIsInstance(result, np.ndarray)
-        self.assertArrayEqual(result, expected)
-
-    def test_freezing_sealevel_point(self):
-        """Test sea point with integral below threshold sets snow level to 0"""
+        plugin.fill_in_high_snow_falling_levels(
+            self.snow_level_data, self.orog, self.highest_wb_int,
+            self.highest_height)
+        self.assertArrayEqual(self.snow_level_data, expected)
+
+    def test_no_fill_if_conditions_not_met(self):
+        """Test it doesn't fill in NaN if the heighest wet bulb integral value
+           is less the than theshold."""
```

Comment:
I think you've just got a tiny typo in "threshold" here.

Reply:
Fixed :)